.github/workflows/tests_gpu_python.yml

```diff
@@ -183,7 +183,7 @@ jobs:
         run: |
           rm -rf build
           PL_BACKEND=lightning_qubit python scripts/configure_pyproject_toml.py || true
-          PL_BACKEND=lightning_qubit SKIP_COMPILATION=True python -m pip install . -vv
+          PL_BACKEND=lightning_qubit python -m pip install . -vv
```

Comment:
We don't need LightningQubit binaries to install LGPU. The installation with `SKIP_COMPILATION=True` should suffice.